analyses/10_mofa/11_easier.R

```diff
@@ -33,7 +33,7 @@ getFeat <- function(dataobj,
     "IFNy",
     "Ayers_expIS",
     "Tcell_inflamed",
-    "RIR",
+    # "RIR",
```

Comment:
@olapuentesantana, the "RIR" score is missing now. The list of available scores is 
```
> colnames(proxies.mat)
 [1] "CYT"            "TLS"            "IFNy"           "Ayers_expIS"   
 [5] "Tcell_inflamed" "Roh_IS"         "Davoli_IS"      "chemokines"    
 [9] "resF_up"        "resF_down"      "resF"      
 ```

Anything I should be selecting instead/in addition?

Reply:
The equivalent for `"RIR"` now should be `"resF_down"`.

```
# List of immune response scores to be computed
  selscores <- c(
    "CYT",
    "Roh_IS",
    "chemokines",
    "Davoli_IS",
    "IFNy",
    "Ayers_expIS",
    "Tcell_inflamed",
    "resF_down",
    "TLS"
  )
```